src/pytorch_lightning/loops/utilities.py

```diff
@@ -220,3 +221,16 @@ def _reset_progress(loop: Loop) -> None:
 def _v1_8_output_format(fx: Callable) -> bool:
     parameters = inspect.signature(fx).parameters
     return "new_format" in parameters and parameters["new_format"].default is True
+
+
+def _set_sampler_epoch(dataloader: DataLoader, epoch: int) -> None:
```

Comment:
added to loops utilities because it is used in loops, but could also be categorized as data utilities.